docs/upgrade/0.5.x_to_0.6.rst

```diff
@@ -0,0 +1,220 @@
+
+Compare the behavior of the old kernel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. |GRUB with advanced options selected| image:: ../images/0.5.x_to_0.6/grub-with-advanced-options-selected.png
+
+Reboot the server in a safe way with ``sudo reboot``. After the BIOS screen,
+you can select a new kernel from the GRUB boot menu by selecting
+**Advanced Options for Ubuntu**, pictured below.
+
+|GRUB with advanced options selected|
+
+Under **Advanced Options**, choose the option with kernel version ``3.14.x-grsec``.
+As before, you may need to edit the kernel options to enter single user
+mode. The boot process should proceed normally. Wait until you get a
+login prompt and log in.
+
+Once you are logged in, check to see if you have network access. If you do, then  
+your instance is having an issue with the 4.x kernel. In that case, we need to 
+temporarily set an older kernel as the default.
+
+Roll back to the old kernel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important:: The older kernel has reached end of life, so it
+  is of critical importance for the near term security of your instance
+  that we work together to resolve any compatibility issues. Rolling back to an 
+  older version is only a stopgap measure to avoid a prolonged outage of your
+  SecureDrop instance.
+
+Inspect the file ``/boot/grub/grub.cfg``. You should find a ``menuentry`` line
+with the same text that you selected during boot, e.g.:
+
+.. code:: sh
+
+  submenu 'Advanced options for Ubuntu'…
+
+    menuentry 'Ubuntu, with Linux 3.14.xx-grsec…
+
+Take note of its position among the other submenu entries (it will most likely 
+be third). Then edit the GRUB configuration:
+
+.. code:: sh
+
+  sudo vim /etc/grub/default
+
+Make a backup of the file or take a note of the current value of 
+``GRUB_DEFAULT`` somewhere, so you can restore the previous behavior easily at a 
+later point.
+
+Once you have done so, set the ``GRUB_DEFAULT`` variable to point to the index 
+of the  menu and submenu. Note that the index starts at 0, so for a typical 
+setup, the line in ``/etc/grub/default`` would look like this:
+
+.. code:: sh
+
+  GRUB_DEFAULT=”1>2”
+
+The “1” means the second entry of the main menu (“Advanced options”),
+the “2” means the third entry of the submenu. Again, update these
+numbers consistent with your configuration. 
+
+
+.. caution:: Ensure that you have chosen the right index for the main menu
+  and the submenu, and double-check that you are beginning the count at 0, not
+  1; otherwise, you may boot into the wrong kernel.
+
+This change still has to be applied to take effect on the next boot:
+
+.. code:: sh
+
+  sudo update-grub2
+
+Now you can reboot into the old, working kernel.
+
+.. code:: sh
+
+  sudo reboot
+
+The server should come up automatically. From here on, you should be
+able to perform all administrative tasks via SSH as usual. If you want
+additional confirmation of the kernel version, the command 
+``uname -a`` should display ``3.14.79-grsec``.
```

Comment:
nit: `uname -r` would specifically return that string, `uname -a` returns a longer string